README.md

```diff
@@ -61,6 +61,21 @@ Then restart the dockerd service:
 sudo snap restart docker.dockerd
 ```
 
+### Docker Snap Aliases
+
+[Snap aliases](https://snapcraft.io/docs/commands-and-aliases) are part of the Snap ecosystem.  
+When installing Docker from the [Snap Store](https://snapcraft.io/), the alias `docker-compose` is automatically created, pointing to `docker.compose`.  
+
+You can verify this by running:  
+
+```console
+$ snap aliases docker
+Command         Alias           Notes
+docker.compose  docker-compose  -
+```  
+
+Although the [migration to Compose V2](https://docs.docker.com/compose/releases/migrate/) deprecates the `docker-compose` command in favor of `docker compose`, the alias still exists for backward compatibility.
```

Comment:
I'd simplify and move this to same section with connections. Example:
> The `docker-compose` [alias](https://snapcraft.io/docs/commands-and-aliases) was set automatically for Compose V1 and remains for backwards-compatiblity. The recommended command-line syntax since Compose V2 is `docker compose` [[ref](https://docs.docker.com/compose/releases/migrate/)].